### PR TITLE
add rumour misinfo sub themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Thematic doc: https://docs.google.com/spreadsheets/d/1dJKooD030TAbHgNgFgUt5A6dD_aP5nz3k1nPyFwpseo/edit#gid=1568850235

### CODE REVIEW

File: code_schemes/s01e01.json [hunk 1]
```diff
@@ -363,6 +363,46 @@
...
+      },
+      {
+        "CodeID": "code-b5594922",
+        "CodeType": "Normal",
+        "DisplayText": "virus description",
+        "NumericValue": 44,
+        "StringValue": "virus_description",
+        "VisibleInCoda": true
+      },
+      {
```
Comment (as2388) [inline code_schemes/s01e01.json:390]: The doc also has negative stigma/anger

File: code_schemes/s01e01.json [hunk 2]
```diff
@@ -363,6 +363,46 @@
         "StringValue": "other_theme",
         "VisibleInCoda": true
       },
+      {
+        "CodeID": "code-ee733b09",
+        "CodeType": "Normal",
+        "DisplayText": "the virus discriminates",
```
Comment (as2388) [inline code_schemes/s01e01.json]: If these are L4 codes do you want to prefix with 'rumour/misinfo' like we did for covid19-ke-urban? I'm not sure, it's up to RDA.

File: code_schemes/s01e01.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/s01e01.json:390]: Thanks for catching this. updated.

File: code_schemes/s01e01.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/s01e01.json]: I agree. We also did this in COVID19-Som. have added rumour/stigma/misinfo prefix

**Review by as2388 — APPROVED** [review]

**Pull request merged**